README.md

```diff
@@ -23,6 +23,8 @@ This workflow is built using [Nextflow](https://www.nextflow.io/) and processes
 4. Coverage calculation by [`mosdepth`](https://github.com/brentp/mosdepth)
 5. Present QC for raw reads ([`MultiQC`](http://multiqc.info/))
 
+![metro map](docs/images/qc_metromap.svg)
```

Comment:
```suggestion
![A high-level diagram of the GRZ QC workflow in a metro map style](docs/images/qc_metromap.png)
```

I would use the PNG for display since SVG can render strangely with some browsers, though we should definitely keep the SVG in the repository too. Also good to have a long description for the image for vision-impaired readers :slightly_smiling_face: